test/message/message.status

```diff
@@ -0,0 +1,18 @@
+prefix message
+
+# To mark a test as flaky, list the test name in the appropriate section
+# below, without ".js", followed by ": PASS,FLAKy". Example:
```

Comment:
+ To mark a test as flaky, list the test name in the appropriate section
+ below, without ".js", followed by ": PASS,FLAKy". Example: